apps/api-documenter/src/yaml/OfficeYamlDocumenter.ts

```diff
@@ -135,4 +136,8 @@ export class OfficeYamlDocumenter extends YamlDocumenter {
   private _fixCodeTicks(text: string): string {
     return Text.replaceAll(text, '\\`', '`');
   }
+
+  private _fixCodeArrows(text: string): string {
+    return Text.replaceAll(text, '=&gt;', '=>');
```

Comment:
>&gt; [](start = 35, length = 4)

Is `&gt;` NOT appearing on the final page as `>`?  If so, how does it get rendered?  That sounds like a bug with the DocFX Markdown renderer that should get reported and fixed.

Reply:
@pgonzal  Yeah, here's a sample: https://review.docs.microsoft.com/en-us/javascript/api/outlook/office.appointmentcompose?branch=AlexJ-DTRefresh

(Remarks section of addFileAttachmentAsync)

Reply:
Ohhh this is because you're using backticks, which DocFX is interpreting as a Markdown code span, whereas AEDoc currently thinks they are just a literal backtick.  By design, API Documenter should have escaped that better.  

But since you're relying on this feature, we won't "fix" the bug.  :-)  Instead let's merge your PR for now, and then address it properly when the TSDoc parser is integrated.

Reply:
For the record, here's the yml line:
`addFileAttachmentAsync(uri: string, attachmentName: string, callback: (result: AsyncResult) =&gt; void): void;`

Reply:
But this line is surrounded by backticks, right?

Reply:
I can add that check. I was just doing the shortcut fix of looking for that => symbol. I can make my change more robust and ensure that only happens within code backticks

Reply:
Nah, this fix seems more narrow and safe